peachjam/js/components/FindDocuments/index.vue

```diff
@@ -639,6 +640,9 @@ export default {
             if (response.ok) {
               this.error = null;
               this.searchInfo = await response.json();
+              if (searchParams) {
+                analytics.trackSiteSearch(searchParams, this.searchInfo.count);
```

Comment:
Matomo lets us push three things:

* the search term
* a search category (optional)
* number of hits

Providing the full searchParams string is too overwhelming. Instead, let's do this:

* search term: use `q` for simple searches, for advanced searches provide all the `search_*` fields; eg `search__title=x&search__content=-y`
* category: use the document type and court facets, if in use, separated by semicolons; eg. `Judgment; High Court` or `High Court` or `Judgment`

Using `params` before it is turned into a string should make this a bit easier.

For GA, just include whatever it supports (probably just the search term)